doc/connectivity/bluetooth/api/mesh/rpr_srv.rst

```diff
@@ -0,0 +1,26 @@
+.. _bluetooth_mesh_models_rpr_srv:
+
+Remote Provisioning Server
+##########################
+
+The Remote Provisioning Server model is a foundation model defined by the Bluetooth
+mesh specification. It is enabled with the
+:kconfig:option:`CONFIG_BT_MESH_RPR_SRV` option.
+
+The Remote Provisioning Server model is introduced in the Bluetooth Mesh Protocol
+Specification version 1.1, and is used to support the functionality of remotely
+provisioning devices into a mesh network.
+
+The Remote Provisioning Server does not have an API of its own, but relies on a
+:ref:`bluetooth_mesh_models_rpr_cli` to control it. The Remote Provisioning Server
+model only accepts messages encrypted with the node's device key.
+
+If present, the Remote Provisioning Server model must be instantiated on the primary
+element. It may also be supported by any secondary element.
```

Comment:
If I read the code correctly we only support single RPR Server instance which can only be instantiated on the primary element: https://github.com/zephyrproject-rtos/zephyr/blob/3e7205876cb5145d6630a7d1bcaf09945009f9f3/subsys/bluetooth/mesh/rpr_srv.c#L1306-L1310. So the second sentence can be removed.